modules/e2fsprogs

```diff
@@ -0,0 +1,32 @@
+modules-$(CONFIG_E2FSPROGS) += e2fsprogs
+
+e2fsprogs_version := 1.47.0
+e2fsprogs_dir := e2fsprogs-$(e2fsprogs_version)
+e2fsprogs_tar := e2fsprogs-$(e2fsprogs_version).tar.xz
+e2fsprogs_url := https://mirrors.edge.kernel.org/pub/linux/kernel/people/tytso/e2fsprogs/v$(e2fsprogs_version)/$(e2fsprogs_tar)
+e2fsprogs_hash := 144af53f2bbd921cef6f8bea88bb9faddca865da3fbc657cc9b4d2001097d5db
+
+# For reproducibility reasons we have to override the exec_prefix
+# and datarootdir on the configure line so that the Makefiles will
+# be generated with the correct paths, but then re-write them when
+# we use the install target so that they will be copied to the correct
+# location.
+e2fsprogs_configure := \
+	$(CROSS_TOOLS) \
+	CFLAGS="-Os"  \
+	./configure \
+	--host $(MUSL_ARCH)-linux-musl \
+	--prefix "/" \
```

Comment:
Use `--prefix ""` - no prefix.  `--prefix "/"` causes funny-looking `//lib`, `//bin`, etc. to appear in various places, which don't harm anything but do look odd, and we shouldn't continue making the same mistake IMO.

Reply:
#1460 created